src/js/game/LevelMVC/LevelPlane.js

```diff
@@ -355,6 +338,20 @@ module.exports = class LevelPlane {
     return posToRefresh;
   }
 
+  positionEquivalence(lhs, rhs) {
+    return (lhs[0] === rhs[0] && lhs[1] === rhs[1]);
+  }
+
+  checkEntityConflit(position) {
+    let captureReturn = false;
+    this.levelModel.controller.levelEntity.entityMap.forEach((workingEntity) => {
```

Comment:
This is a sign that `levelEntity` should probably live on `levelModel` instead of controller.  Not blocking this change.

Reply:
Please re-write with `.some()` or `.any()`